airbyte-integrations/connectors/source-postgres/src/test-integration/java/io/airbyte/integrations/io/airbyte/integration_tests/sources/PostgresSourceDatatypeTest.java

```diff
@@ -221,48 +224,43 @@ protected void initTests() {
                 "128.1.0.0/16", "2001:4f8:3:ba::/64")
             .build());
 
-    // JdbcUtils-> DATE_FORMAT is set as ""yyyy-MM-dd'T'HH:mm:ss'Z'"" so it doesnt suppose to handle BC
-    // dates
     addDataTypeTestData(
         TestDataHolder.builder()
-            .sourceType("date")
+            .sourceType("circle")
             .airbyteType(JsonSchemaPrimitive.STRING)
-            .addInsertValues("'1999-01-08'", "null") // "'199-10-10 BC'"
-            .addExpectedValues("1999-01-08T00:00:00Z", null) // , "199-10-10 BC")
-            .build());
-
-    // Values "'-Infinity'", "'Infinity'", "'Nan'" will not be parsed due to:
-    // JdbcUtils -> setJsonField contains:
-    // case FLOAT, DOUBLE -> o.put(columnName, nullIfInvalid(() -> r.getDouble(i), Double::isFinite));
-    // https://github.com/airbytehq/airbyte/issues/7871
-    addDataTypeTestData(
-        TestDataHolder.builder()
-            .sourceType("float8")
-            .airbyteType(JsonSchemaPrimitive.NUMBER)
-            .addInsertValues("'123'", "'1234567890.1234567'", "null")
-            .addExpectedValues("123.0", "1.2345678901234567E9", null)
+            .addInsertValues("'(5,7),10'", "'(0,0),0'", "'(-10,-4),10'", "null")
+            .addExpectedValues("<(5,7),10>", "<(0,0),0>", "<(-10,-4),10>", null)
             .build());
 
+    // DataTypeUtils#DATE_FORMAT is set as "yyyy-MM-dd'T'HH:mm:ss'Z'", so currently the Postgres source
+    // returns a date value as a datetime. It cannot handle BC dates (e.g. 199-10-10 BC).
     addDataTypeTestData(
         TestDataHolder.builder()
-            .sourceType("double_precision")
-            .fullSourceDataType("double precision")
-            .airbyteType(JsonSchemaPrimitive.NUMBER)
-            .addInsertValues("'123'", "'1234567890.1234567'", "null")
-            .addExpectedValues("123.0", "1.2345678901234567E9", null)
+            .sourceType("date")
+            .airbyteType(JsonSchemaPrimitive.STRING)
+            .addInsertValues("'1999-01-08'", "null")
+            .addExpectedValues("1999-01-08T00:00:00Z", null)
```

Comment:
I reverted the change that returns a pure date (without the time portion), because I have not verified that this is a safe change.